target/common/common.mk

```diff
@@ -86,7 +87,13 @@ VLT_FLAGS    += -Wno-UNSIGNED
 VLT_FLAGS    += -Wno-UNOPTFLAT
 VLT_FLAGS    += -Wno-fatal
 VLT_FLAGS    += --unroll-count 1024
+ifneq ($(VLT_NUM_THREADS), 1)
+	VLT_FLAGS	 += --threads $(VLT_NUM_THREADS)
+endif
 VLT_CFLAGS   += -std=c++14 -pthread
+ifneq ($(VLT_NUM_THREADS), 1)
+	VLT_CFLAGS	 += -DVL_THREADED=1
+endif
```

Comment:
Couldn't find much information on this, but perhaps we can also always add this flag, and remove the if statement. I guess it shouldn't break the single-threaded case.

Reply:
Information is sadly very sparse on the flag and I only found it referenced through GitHub issues such as https://github.com/verilator/verilator/issues/3668#issuecomment-1288989145
I had to add it as I was getting compilation errors as verilator requires it to be defined when using multiple threads. 
I added a comment to clarify.